addons/website/views/snippets/s_card_offset.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_card_offset" name="Card Offset">
+    <section class="s_card_offset o_cc o_cc1 pt64 pb64">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="9">
+                <div class="o_grid_item o_grid_item_image g-height-8 g-col-lg-12 col-lg-12 rounded d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 1; grid-area: 2 / 1 / 10 / 13; --grid-item-padding-x: 0px; --grid-item-padding-y: 0px; border-radius: 6.4px !important;">
+                    <img src="/web/image/website.s_card_offset_default_image" class="img img-fluid mx-auto" alt=""/>
+                </div>
+                <div class="o_grid_item o_colored_level o_cc o_cc1 g-height-7 g-col-lg-5 col-lg-5 col-10 offset-1 offset-lg-0 rounded justify-content-center shadow" style="z-index: 2; grid-area: 1 / 7 / 8 / 12; --grid-item-padding-x: 32px; box-shadow: rgba(0, 0, 0, 0.15) 0px 0px 0px 10px !important;">
```

Comment:
- You don't need to define `o_colored_level` manually here. ;)
- Also, same nitpicking than the previous review
```suggestion
                <div class="o_grid_item o_cc o_cc1 g-height-7 g-col-lg-5 col-lg-5 col-10 offset-1 offset-lg-0 justify-content-center rounded shadow" style="z-index: 2; grid-area: 1 / 7 / 8 / 12; --grid-item-padding-x: 32px; box-shadow: rgba(0, 0, 0, 0.15) 0px 0px 0px 10px !important;">
```


- Also, what do you think about adding the same vertical padding, so the mobile version would be more consistent.
```suggestion
                <div class="o_grid_item o_colored_level o_cc o_cc1 g-height-7 g-col-lg-5 col-lg-5 col-10 offset-1 offset-lg-0 rounded justify-content-center shadow" style="z-index: 2; grid-area: 1 / 7 / 8 / 12; --grid-item-padding-x: 32px; --grid-item-padding-y: 32px; box-shadow: rgba(0, 0, 0, 0.15) 0px 0px 0px 10px !important;">
```
| Before | After |
|--------|--------|
| ![Capture d’écran 2024-08-23 à 15 49 04](https://github.com/user-attachments/assets/fbba6b16-cb52-4fe1-b025-4fa1436abade) | ![Capture d’écran 2024-08-23 à 15 48 51](https://github.com/user-attachments/assets/75b0cf4c-f280-4291-9cb1-856c0b5b3b07) |